### PR TITLE
fetch all itmes in items.txt

### PR DESCRIPTION
**Pull request by dieDeMiguel**
This route now brings the content of items.txt

### CODE REVIEW

File: routes/index.js [hunk 1]
```diff
@@ -10,8 +10,20 @@ router.get('/', function(req, res, next) {
 
 // GET objects array complete
 router.get('/items', function(req, res, next) {
-  res.send('Here you will find the array parsed');
+  
+    try {
+      fs.readFile('items.txt', (err, data) => {
```
Comment (charlitos85) [inline routes/index.js:15]: move items.txt to a constants

Comment (charlitos85) [reply routes/index.js:15]: dont used data as callback, please a descriptible name

File: routes/index.js [hunk 2]
```diff
@@ -10,8 +10,20 @@ router.get('/', function(req, res, next) {
 
 // GET objects array complete
 router.get('/items', function(req, res, next) {
-  res.send('Here you will find the array parsed');
+  
+    try {
```
Comment (charlitos85) [inline routes/index.js:14]: this is not required in this point

File: routes/index.js (hunk 1 above)
Comment (dieDeMiguel) [reply routes/index.js]: done

File: routes/index.js (hunk 2 above)
Comment (dieDeMiguel) [reply routes/index.js:14]: done

**Comment by dieDeMiguel** [comment]
Unnecessary branch

**Pull request closed without merging**